lib/compare_task.rb

```diff
@@ -27,7 +27,7 @@ def compare(description1, description2, scopes, options = {})
   end
 
   def render_html_comparison(description1, description2, scopes, options)
-    LocalSystem.validate_existence_of_package("xdg-utils")
+    LocalSystem.validate_existence_of_packages(["xdg-utils"])
```

Comment:
It would be better to allow to just pass a name and not require to do that as array even if it's only one package.

The `validate_existence_of_packages` method could just internally convert the parameter to an array if it's passed as string.